build.sbt

```diff
@@ -51,14 +55,15 @@ lazy val commonSettings = Seq(
   Compile / doc / scalacOptions ++= Opts.doc.title("epimetheus"),
 
   libraryDependencies ++= Seq(
-    "io.prometheus"               % "simpleclient"                % prometheusV,
-    "io.prometheus"               % "simpleclient_common"         % prometheusV,
-    "io.prometheus"               % "simpleclient_hotspot"        % prometheusV,
+    "io.prometheus"               % "prometheus-metrics-core"                % prometheusV,
+    "io.prometheus"               % "prometheus-metrics-instrumentation-jvm" % prometheusV,
+    "io.prometheus"               % "prometheus-metrics-exposition-formats"  % prometheusV,
```

Comment:
We need this `exposition-formats` package to be able to expose `PrometheusRegistry#write004` and `PrometheusRegistry#writeOpenMetrics100`.

@ChristopherDavenport do you think it's worth it? Or should we drop the methods?

Reply:
Doesn't something have to write those out so the scraper can access the metric values? And if so, it's an effectual operation, so it would be good to have those methods here.

Reply:
Agreed.

Reply:
My (perhaps naive) expectation is that the systems that will be scraping metrics won't be doing this via this interface. So it is thinkable that we'd drop these methods (`write004` and `writeOpenMetrics100`), just as we've dropped the `get` methods (because they've disappeared from the underlying Java interfaces).

But there was this workaround available to keep the methods around and I wanted to check with you :+1: